src/components/Header/Header.tsx

```diff
@@ -0,0 +1,78 @@
+import configPromise from '@payload-config'
+import { getPayload } from 'payload'
+
+import { draftMode } from 'next/headers'
+import Link from 'next/link'
+import { ImageMedia } from '../Media/ImageMedia'
+import { DesktopNav } from './DesktopNav.client'
+import { MobileNav } from './MobileNav.client'
+import { getTopLevelNavItems } from './utils'
+
+export async function Header({ center }: { center?: string }) {
+  const { isEnabled: draft } = await draftMode()
+  const payload = await getPayload({ config: configPromise })
+
+  const media = await payload.find({
+    collection: 'brands',
+    depth: 10,
+    where: {
+      'tenant.slug': {
+        equals: center,
+      },
+    },
+  })
+
+  if (media.docs.length < 1) {
+    payload.logger.error(`Brand for tenant ${center} missing`)
+  }
+
+  const banner = media.docs[0]?.banner
+
+  if (media.docs.length > 0 && typeof banner !== 'object') {
+    payload.logger.error(`Banner for tenant ${center} missing`)
```

Comment:
I wonder - can we also `throw` something here (and generally in all cases where we're doing this defensive log), so that `pnpm build` errors out and we catch this in CI?